pkgs/development/python-modules/executing/default.nix

```diff
@@ -25,10 +27,10 @@ buildPythonPackage rec {
     export SETUPTOOLS_SCM_PRETEND_VERSION="${version}"
   '';
 
-  # Tests appear to run fine (Ran 22 tests in 4.076s) with setuptoolsCheckPhase
-  # but crash with pytestCheckHook
   checkInputs = [
+    pytestCheckHook
     asttokens
+    littleutils
   ];
```

Comment:
Per [this line in setup.cfg](https://github.com/alexmojaki/executing/blob/master/setup.cfg#L37), let's add:

```
] ++ lib.optionals (pythonAtLeast "3.11") [
  rich
];
```